dts/bindings/serial/infineon,cat1-uart.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2018, Cypress
+# Copyright (c) 2020, ATL Electronics
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Infineon CAT1 UART
+
+compatible: "infineon,cat1-uart"
+
+include: [uart-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  peripheral-id:
+    type: int
+    description: peripheral ID
+    required: true
+
+  pinctrl-0:
+    description: |
+      PORT pin configuration for TX, RX, RTS, CTS signals.
+      We expect that the phandles will reference pinctrl nodes. These
+      nodes will have a nodelabel that matches the Infineon SoC Pinctrl
+      defines and be of the form p<port>_<pin><peripheral inst>_<signal>.
+
+      Examples
+         use TX, RX
+         pinctrl-0 = <&p5_1_scb5_uart_tx &p5_0_scb5_uart_rx>;
+
+         use RX only
+         pinctrl-0 = <&p5_0_scb5_uart_rx>;
+
+         use TX, RX, RTS, CTS
+         pinctrl-0 = <&p3_1_scb2_uart_tx &p3_0_scb2_uart_rx
+                      &p3_2_scb2_uart_rts &p3_3_scb2_uart_cts>;
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  stop-bits:
+    type: string
+    description: |
+      The number of stop bits, refer to uart_config_stop_bits
+    enum:
+      - "0.5"
+      - "1"
+      - "1.5"
+      - "2"
+    default: "1"
```

Comment:
remove default here.

Reply:
removed. Updated uart_ifx_cat1.c driver to handle default values for stop_bits/data_bits:
```
.#define INFINEON_CAT1_UART_INIT(n)							     \
...
	static struct ifx_cat1_uart_config ifx_cat1_uart##n##_cfg = {			     \
	   .dt_cfg.stop_bits = DT_INST_ENUM_IDX_OR(n, stop_bits, UART_CFG_STOP_BITS_1), \
	   .dt_cfg.data_bits = DT_INST_ENUM_IDX_OR(n, data_bits, UART_CFG_DATA_BITS_8), \
....
```